.github/workflows/build.yaml

```diff
@@ -43,15 +43,18 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v2
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
The latest version of this GitHub action is [v3](https://github.com/docker/setup-buildx-action/releases/tag/v3.0.0). Do you prefer to upgrade from v2?